cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -313,7 +325,9 @@ data:
   Corefile: |
     .:53 {
         errors
-        health
+        health {
+           lameduck 12s
```

Comment:
is 12s the timeout for the health check in this case?
the timeout for CP components is 15s, so we may want to match that.

Reply:
Actually - as I was describing the reasoning behind this, I realized that a timeout of 5 seconds should be all that is necessary.  When picking 12s, I was conflating the issue with the readiness/health check periods, which I don't think actually come into play.  The function of lameduck is to finish processing in flight queries before shutting down.  A lameduck of longer than 5 would typically be pointless, since most clients have a default timeout of 5 seconds (and thus would have stopped listening for a response after then).

Reply:
yep, 5 seems good if that is sufficient.

Reply:
@rajansandeep do you agree with the change to 5 seconds?

Reply:
Yes, I agree. I'll push a commit to reflect those changes.